tests/generic_views/models.py

```diff
@@ -30,6 +30,14 @@ def __str__(self):
         return self.name
 
 
+class Series(models.Model):
+    name = models.CharField(max_length=100)
+    pubdate = models.DateField()
+
+    def __str__(self):
```

Comment:
Is this used? Can an existing model be reused? We try to avoid adding new models if possible since that complicates and slows the tests.